core/db_classes/EE_Base_Class.class.php

```diff
@@ -160,112 +209,90 @@ protected function __construct($fieldValues = [], $by_db = false, $timezone = ''
                 );
             }
         }
-        $this->_timezone = EEH_DTT_Helper::get_valid_timezone_string($timezone);
-        if (! empty($date_formats) && is_array($date_formats)) {
-            list($this->_dt_frmt, $this->_tm_frmt) = $date_formats;
-        } else {
-            // set default formats for date and time
-            $this->_dt_frmt = (string)get_option('date_format', 'Y-m-d');
-            $this->_tm_frmt = (string)get_option('time_format', 'g:i a');
-        }
-        // if db model is instantiating
-        if ($by_db) {
-            // client code has indicated these field values are from the database
-            foreach ($model_fields as $fieldName => $field) {
+    }
+
+    /**
+     * @param array $model_fields
+     * @param bool  $set_from_db
+     * @throws EE_Error
+     * @throws ReflectionException
+     * @since $VID:$
+     */
+    private function setFieldValues(array $model_fields, $set_from_db)
+    {
+        foreach ($model_fields as $fieldName => $field) {
+            // if db model is instantiating
+            if ($set_from_db) {
+                // client code has indicated these field values are from the database
```

Comment:
I'm currently only following PSR standards for new methods (although renaming would essentially be creating a new method too) but am not going that path just yet.

We're planning on setting up a mono-repo for ALL PHP development that will work similar to Barista, where ALL development happens on that single repo and then the finished production ready code gets deployed to wherever it is needed. This means that EE core as well as ALL add-ons will be under the same version control which will allow for a much easier process when it comes to making changes that span across multiple plugins, such as when you rename methods. Once we get PHP development centralized I will consider making some bigger changes because it will be possible to control the entire codebase.